promtail/imageroot/promtail.service

```
@@ -0,0 +1,31 @@
+[Unit]
+Description=Promtail logs collector for Loki
+
+[Service]
+Type=forking
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/var/lib/nethserver/%N/state/environment
+Restart=on-failure
+ExecStartPre=/bin/rm -f %t/%N.pid %t/%N.cid
+ExecStart=/usr/bin/podman run \
+    --conmon-pidfile %t/%N.pid \
+    --cidfile %t/%N.cid \
+    --cgroups=no-conmon \
+    --replace \
+    --log-opt=tag=promtail \
+    --name %N \
+    -d \
+    -e LOKI_URL=${LOKI_URL} \
+    -v /var/log/journal/:/var/log/journal/ \
+    -v /etc/machine-id:/etc/machine-id \
+    -v ./promtail.yaml:/etc/promtail/config.yml \
+    docker.io/grafana/promtail:2.2.1 \
```

When we'll address container upgrades a variable from the environment might help, what do you think?

```suggestion
    ${PROMTAIL_IMAGE} \
```